src/rgw/rgw_admin.cc

```diff
@@ -5834,7 +5834,9 @@ int main(int argc, const char **argv)
       return -r;
     }
 
-    encode_json("status", status, formatter);
+    for (auto s: status) {
+      encode_json("status", s, formatter);
```

Comment:
@oritwas how does the output here look like? Should we open a formatter array section?